scss/standalone/patterns_tab-buttons.scss

```diff
@@ -0,0 +1,5 @@
+@import '../vanilla';
+@include vf-base;
+
+@include vf-p-tabs;
+@include vf-p-icon-info;
```

Comment:
It would be good to clearly separate what is a source of given component styles and what is just needed for the examples.

Usually I was grouping them properly and commenting in code - there should be some examples in other standalone stylesheets.